client/src/temp/tripViewData.json

```diff
@@ -0,0 +1,152 @@
+[
+  {
+    "color": "bg-red-400",
```

Comment:
Can we make the cards more modern and sleek? Pastel colors feel a bit dry

This card is from the MUI website, I like the slight gradient and box shadow
<img width="392" alt="Screenshot 2023-06-04 at 10 54 11 AM" src="https://github.com/cpsc455-bugstorm/TravelersTea/assets/36722714/7f6a697a-2f07-4e69-8c4b-6d92d2bbe663">

Reply:
Sounds good :) I will look into it.

Reply:
that does look nice

Reply:
Done